### DISCUSSION TITLE
Search multiple Zotero libraries/groups simultaneously

### DISCUSSION THREAD

**Discussion by pcuellar** [aidenlx/obsidian-zotero#508] [Ideas]
First of all, great work on this plugin.

I am trying to add markdown citations _(a-la_ citations plugin) from different libraries but I seem to only be able to choose from one library at a time. I can add simple APA citations via Zotero Integration but I cannot transform those with Pandoc upon export. I it possible to support all libraries instead of just one at a time?

**Comment by FeralFlora** [comment]
> I can add simple APA citations via Zotero Integration but I cannot transform those with Pandoc upon export. I it possible to support all libraries instead of just one at a time?

Just a FYI, you can also create a citation format with Pandoc citations in Zotero Integration. But of course, it would be better if you had the option to autocomplete from all libraries.

**Comment by aidenlx** [comment]
supported in v2.1.0-beta.3
